src/AuthenticationPortal.Contracts/Errors/CustomErrorCodes.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Globalization;
+using System.Reflection;
+using System.Resources;
+
+namespace AuthenticationPortal.Contracts
+{
+    public static class CustomErrorCodes
```

Comment:
update the class name to something appropriate , it's a helper class

Reply:
Done